testdata/p4_16_samples_outputs/psa-dpdk-table-key-consolidation-mixed-keys-6.p4.spec

```diff
@@ -69,6 +69,7 @@ header ethernet instanceof ethernet_t
 header ipv4 instanceof ipv4_t
 header tcp instanceof tcp_t
 
+;oldname:NoAction_1
```

Comment:
What happens if a P4 developer writes an action like this in their code, which is perfectly legal, although perhaps unusual:
```
action NoAction_5 () {
     // code here that does something that is NOT a no-op
}
```

Reply:
thanks, fixed it now, it will be preserved now.